edx_sga/sga.py

```diff
@@ -1162,6 +1164,35 @@ def _user_image_url(self, user):
         image_url = "{}{}".format(base_url, profile_image_url)
         return image_url
 
+    def get_team_name(self, username):
+
+        if not self.team_view:
+            return None
+
+        runtime = self.xmodule_runtime
+        user = runtime.service(self, 'user').get_current_user()
+        course_id = runtime.course_id
+
+        xblock_settings = self.get_xblock_settings()
+
+        try:
```

Comment:
Does it makes sense to extract this logic in another method??? then we could use it in any other method that needs to validate the same.

Reply:
Check